doc/guides/gps_setup.md

```diff
@@ -0,0 +1,57 @@
+
+Having the GPS constellation computers within a few blocks of each other is what people commonly do but is not required. So long as at least four GPS hosts can respond to the requester the request will be fulfilled.
+
+GPS only works with wireless modems, ender modems count as wireless. It doesn't matter what side you attach them to, just that each computer in the constellation has one.
+
+:::note Ender modems
+You might be aware that ender modems have a very large range, this makes them very useful for setting up GPS hosts. If you do this then you will likely only need one GPS constellation for the whole dimension (in vanilla this will mean that you'd need three constellations, one for the Overworld, one for the Nether, and one for the End).
+
+Also, don't worry about an ender modem's ability to work across dimensions, the GPS API is smart enough to ignore constellations that are not in its dimension.
+:::
+
+## Configureing the constellation
+To provide computers (or more commonly, turtles) their location, every GPS host needs to know its own location. GPS host can get this from other constellations, but we are going to input it manually, which means that we are going to make a startup file.
+
+It doesn't matter what kind of startup file you use, so long as it's a valid one; the contents of the file is going to be the same.
+
+Here's a template for the startup file's contents:
+```lua
+shell.run("gps", "host", x, y, z)
+```
+Where `x`, `y`, and `z` are the respective coordinates of the CC computer. The easiest way to get the computer's coordinates is to look at it and press `F3`, its coordinates can then be found on the right side of the screen as `Targeted Block`.
+
+:::note Why use MC's coordinates?
+CC doesn't care if you use Minecraft's coordinate system, so long as all of the GPS hosts with overlapping ranges use the same reference point (requesting computers will get confused if hosts have different reference points). However, using MC's coordinate system does provide a nice standard to adopt server-wide. It also is consistaint with how command computers get their location, they use MC's command system to get their block which returns that in MC's coordinate system.
+:::
+
+:::note Modem messages come from the computer's position, not the modem's
+Wireless modems transmit from the block that they are attached to *not* the block space that they occupy, the coordinates that you input into your GPS host should be the position of the computer and not the position of the modem.
+:::
+
+## A brief look at how the GPS API works in CC
+Except for command computers and user code overrides, ComputerCraft mimics real-life GPS; computers trilaterate themselves using signals with known sender positions (which is why we have to tell the GPS hosts where they are). This means that the average CC computer will need a wireless modem and be in range of a GPS constellation.
```

Comment:
What do you mean by "the average CC computer"? I assume you're referring to non-command ones here?

Reply:
Yeah, I'd just leave that off then.

I'm not really sure about this section in its current state. Right now it really boils down to "it's Trilateration!". I don't know if we want a couple of sentences explanation, or just drop it entirely.